src/main/java/org/jabref/model/entry/field/InternalField.java

```diff
@@ -12,7 +12,6 @@
 public enum InternalField implements Field {
     OWNER("owner"),
     TIMESTAMP("timestamp", FieldProperty.DATE),
-    GROUPS("groups"),
```

Comment:
Wasn't the samentics of "InternalField" that is a field introduced by JabRef? (The class should be ranamed to `JabRefSuperPowerFields` (when going extreme).

`OWNER` and `TIMESTAMP` are "normal" BibTeX fields - just introdced by JabRef.

If we move `GROUPS` to `StandardField`, we should move the other two, too.

@stefan-kolb What's your take on this?

Reply:
Probably `JabrefCustomFields`, `CustomFields`?

Reply:
I now moved all three (group, owner, timestamp) to standard fields. They need to be grouped with the other fields in one enum to use the power of the enummap. Now the "StandardField" refers to a field that is a normal field for JabRef.